docs/indicator-configuration.md

```diff
@@ -608,6 +608,11 @@ standalone: true
 
 In this case you may also want to control the URL of the indicator. You can do this with the `permalink` metadata field (see above).
 
+If you want to create a page for your standalone indicators you can do so in the `_pages` folder of your site repository. For more details on how to create pages, see the [create_pages guidance](https://open-sdg.readthedocs.io/en/latest/tutorials/create-pages/). It is recommended the following configuration is used on your created page:
```

Comment:
@otis-bath For a bit more clarification maybe "If you want to create a listing page"?

Reply:
@brockfanning - thanks Brock have changed this